neuror/main.py

```diff
@@ -634,6 +641,7 @@ def repair(inputfile: Path,
         plot_file: the filename of the plot
         repair_flags: a dict of flags where key is a RepairType and value is whether
             it should be repaired or not. If not provided, all types will be repaired.
+        apical_point: 3d vector for apical point posiiton, if known
```

Comment:
You can add in both docstrings that else, the automatic apical detection is used